### PR TITLE
Prototypes/case study page

### PR DESCRIPTION
**Pull request by AriannaChau**
## Overview

This PR adds a new Case Study Listings prototype. It adds a possible layout for the list of projects and imagery ideas we could go with.

## Screenshots

**Large screens**
localhost_6006_iframe html_id=prototypes-case-study-listings--example args= viewMode=story copy

localhost_6006_iframe html_id=prototypes-case-study-listings--example args= viewMode=story (1)

**Small screens**
localhost_6006_iframe html_id=prototypes-case-study-listings--example args= viewMode=story copy 2

### Updated Screenshots

**Book imagery and text**
localhost_6006_iframe html_id=prototypes-case-study-listings--example args= viewMode=story copy

**Open source section with imagery**
localhost_6006_iframe html_id=prototypes-case-study-listings--example args= viewMode=story copy 3

## Testing

1. [Deploy link](https://deploy-preview-1331--cloudfour-patterns.netlify.app/?path=/story/prototypes-case-study-listings--example)

---

- Fixes https://github.com/cloudfour/cloudfour.com-patterns/issues/1295

### CODE REVIEW

**Review by tylersticka — COMMENTED** [review]
@AriannaChau Can you ping me for re-review once you've added treatments for open source projects and books? (I see a "Books" heading but it doesn't look like that content has been tailored for that)

**Comment by tylersticka** [comment]
@AriannaChau I think this looks good!

Some suggestions:

- Can we try the book image with the one used by [A Book Apart](https://abookapart.com/products/progressive-web-apps)? I think it might look nice here.
- Can we differentiate the different sections a bit more? They're so close together and the headings are so similar in size that they're all kind of bleeding together for me right now.

**Review by tylersticka — COMMENTED** [review]
@AriannaChau I think this looks good!

Some suggestions:

    Can we try the book image with the one used by A Book Apart? I think it might look nice here.
    Can we differentiate the different sections a bit more? They're so close together and the headings are so similar in size that they're all kind of bleeding together for me right now.

File: src/prototypes/case-study-listings/example/example.twig [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+        </article>
+      </div>
+    </div>
+  </div>
+
+  <div class="wrapper book-container">
+    <div class="o-container o-container--pad">
+      <div class="o-container__content">
+        <div class="title">
+          <h1>Books</h1>
```
Comment (tylersticka) [inline src/prototypes/case-study-listings/example/example.twig:105]:
```suggestion
          <h1>Books we’ve written</h1>
```

File: src/prototypes/case-study-listings/example/example.twig [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+          </article>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="wrapper o-container o-container--pad">
+    <div class="o-container__content">
+      <div class="title">
+        <h1>Open Source</h1>
```
Comment (tylersticka) [inline src/prototypes/case-study-listings/example/example.twig:141]:
```suggestion
        <h1>Our open source projects</h1>
```

**Review by tylersticka — COMMENTED** [review]
I like the alternating background. Some more suggestions...

1. The top case studies section feels like it could use some more top padding compared to what's below.
2. The headings feel like they should be closer to the content than the preceding container.
3. Jason has written two books. How would it feel to have two books in that section? 
4. Should we include a link to our GitHub in relation to our open source projects?

**Review by tylersticka — COMMENTED** [review]
The padding in sections feels inconsistent. It starts out nice and roomy but then gets smaller as the page continues.

The book images kind of dominate more than I expected them to. This won't block the merging of the PR, but I'd love to see alternate treatments for the open source projects and books that make them a bit more secondary to the case studies in some ways (more similar to how Labs projects are shown today). Maybe even keeping the headings in the first column at large sizes?

**Comment by AriannaChau** [comment]
@tylersticka, I've gone ahead and adjusted the padding. I'm looking into alternative ways to handle the books and open source sections, but requesting your re-review while I do that to keep the ball rolling.

**Review by tylersticka — APPROVED** [review]

**Pull request merged**